build.sbt

```diff
@@ -271,12 +260,13 @@ lazy val zincCompileCore = (project in internalPath / "zinc-compile-core")
 //   format from which Java sources are generated by the sbt-contraband plugin.
 lazy val compilerInterface = (project in internalPath / "compiler-interface")
   .enablePlugins(ContrabandPlugin)
-  .disablePlugins(com.typesafe.sbt.SbtScalariform)
   .settings(
     minimalSettings,
     // javaOnlySettings,
     name := "Compiler Interface",
-    crossScalaVersions := Seq(scala212),
+    // Use the smallest Scala version in the compilerBridgeScalaVersions
```

Comment:
It would be good to make this comment more descriptive and explain why this is the case so that future readers understand that the bridge has to be as much compatible as possible.